### PR TITLE
client events

### PR DESCRIPTION
**Pull request by simran8072**
Added cards
Included an "Add event" functionality
Routed the Event page to the Event Form page

### CODE REVIEW

**Comment by shifuchen98** [comment]
Please make sure all files are well-formatted (you can run `npm run lint` to batch format all files), and remove any piece of commented code if you don't need it.

File: src/components/ClientEvents.vue [hunk 1]
```diff
@@ -1,6 +1,113 @@
...
+              <h1>Welcome Email with steps</h1>
+              <!-- Event name -->
+              <div class="field">
+                <p>Last updated on:</p>
+                <input v-model="lastUpdated" type="date" style="width:40%" />
+                <button class="primary" style="margin-left:5px">Save</button>
+              </div>
+              <div>
+                <label> Status: </label>
+                <select v-model="selected" style="width:40%">
```
Comment (shifuchen98) [inline src/components/ClientEvents.vue:24]: Maybe give a default value to each `<select>` to indicate the status each client starts with.

File: src/components/ClientEvents.vue [hunk 2]
```diff
@@ -1,6 +1,113 @@
...
+    <div>
+      <div class="columns">
+        <div class="column">
+          <div class="field">
+            <button class="primary" @click="$router.push('/events/add')">
+              Add Event
+            </button>
+          </div>
+
+          <div class="card">
```
Comment (shifuchen98) [inline src/components/ClientEvents.vue:13]: Fitting cards within a card doesn't look very pretty. Consider dividing different parts with `<section>`.

File: src/components/EventForm.vue [hunk 3]
```diff
@@ -0,0 +1,22 @@
+<template>
+  <section class="fields">
+    <h1>ADDING EVENT</h1>
```
Comment (shifuchen98) [inline src/components/EventForm.vue:3]: Use `Title Case`.

File: src/components/ClientEvents.vue [hunk 4]
```diff
@@ -1,6 +1,113 @@
...
+            <div class="fields">
+              <h1>Welcome Email with steps</h1>
+              <!-- Event name -->
+              <div class="field">
+                <p>Last updated on:</p>
+                <input v-model="lastUpdated" type="date" style="width:40%" />
```
Comment (shifuchen98) [inline src/components/ClientEvents.vue:19]: Fields referenced in `v-model` should be present in `data` as well.

File: src/components/ClientEvents.vue [hunk 5]
```diff
@@ -1,6 +1,113 @@
...
+                <button class="primary" style="margin-left:5px">Save</button>
+              </div>
+              <div>
+                <label> Status: </label>
```
Comment (shifuchen98) [inline src/components/ClientEvents.vue:23]: Remove extra whitespace.

File: src/components/ClientEvents.vue [hunk 6]
```diff
@@ -1,6 +1,113 @@
 <template>
   <section class="fields">
-    <h1>Events</h1>
+    <h1>Client Milestones</h1>
+    <div>
+      <div class="columns">
+        <div class="column">
+          <div class="field">
+            <button class="primary" @click="$router.push('/events/add')">
```
Comment (shifuchen98) [inline src/components/ClientEvents.vue:8]: `/` at the beginning of a route means relative to root, so either remove it or write everything out like `/client/${$route.params.id}/events/add`.

File: src/router/index.js [hunk 7]
```diff
@@ -108,6 +109,12 @@ const routes = [
             path: "events",
             component: () => import("../components/ClientEvents.vue")
           },
+
+          {
+            path: "/events/add",
```
Comment (shifuchen98) [inline src/router/index.js:114]: Remove the `/` at the beginning.

File: src/components/ClientEvents.vue (hunk 1 above)
Comment (skmemon) [reply src/components/ClientEvents.vue:24]: Added the <selected> attribute to "Not Done" status, however not updating when we run the code

File: src/components/ClientEvents.vue (hunk 2 above)
Comment (skmemon) [reply src/components/ClientEvents.vue:13]: We named the class "Card" however, there are no cards within a card, each is a separate card.

File: src/components/ClientEvents.vue (hunk 1 above)
Comment (shifuchen98) [reply src/components/ClientEvents.vue:24]: So you need to have a spot in `data` for storing the current state of each `<select>`, and bind `<select>` with that data. At the same time, assign a value to each `<option>` (the content wrapped within `<option></option>` doesn't automatically become its value). When the item being selected is updated, update the value in `data`.

Comment (shifuchen98) [reply src/components/ClientEvents.vue:24]: Take a look: https://vuejs.org/v2/guide/forms.html#Select

Comment (shifuchen98) [reply src/components/ClientEvents.vue:24]: So you need to have a field in `data` for storing the current state of each `<select>`, and bind `<select>` with that field using `v-model`.

Take a look: https://vuejs.org/v2/guide/forms.html#Select

**Pull request merged**